<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>584</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>641</width>
    <height>520</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="plotWindows" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openB">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="plotBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Plot setting</string>
   </property>
   <widget class="QPushButton" name="Pgen">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Regenerate</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Max. V</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Vmax"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Min. V</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Vmin"/>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>bwr</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>jet</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>rainbow</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="MEPBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>241</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>MEP searching</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="outBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>661</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
